tests/test_sns/test_topics_boto3.py

```diff
@@ -522,6 +522,16 @@ def test_untag_resource_error():
     ).should.throw(ClientError, "Resource does not exist")
 
 
+@mock_sns
+def test_create_fifo_topic():
+    conn = boto3.client("sns", region_name="us-east-1")
+    response = conn.create_topic(
+        Name="test_topic.fifo", Attributes={"FifoTopic": "true"}
+    )
+
+    assert "TopicArn" in response
+
+
```

Comment:
Can we add a negative test as well? Just to assert that the correct error is thrown when creating a FIFO-queue without `.fifo` suffic